### ISSUE TITLE
Unable to fetch your Twitch follows

### ISSUE THREAD

**Issue by Njegosh** [vinszent/gnome-twitch#403]
Got this message today:
`Unable to fetch all followed channels because: Unable to read JSON member with name '_total' because: The member “_total” is not defined in the object at the current position.`

Using the Flatpak package.

**Comment by ski-freak** [comment]
Same, also crashes when loading a stream.
Please help

**Comment by Njegosh** [comment]
> Same, also crashes when loading a stream.
> Please help

I have only recently seen this https://github.com/vinszent/gnome-twitch/issues/402 so I do not expect a fix or help any time soon

**Comment by ski-freak** [comment]
I think this project was discontinued because of the twitch api stuff.

**Comment by vonschutter** [comment]
YEP Same here: Unable to fetch all followed channels because: Unable to read JSON member with name '_total' because: The member “_total” is not defined in the object at the current position.
